packages/vite/src/node/index.ts

```diff
@@ -207,11 +207,7 @@ export type {
 export type { Connect } from 'dep-types/connect'
 export type { WebSocket, WebSocketAlias } from 'dep-types/ws'
 export type { HttpProxy } from 'dep-types/http-proxy'
-export type {
-  FSWatcher,
-  WatchOptions,
-  AwaitWriteFinishOptions,
```

Comment:
I removed this export because initially it wasn't exported by `chokidar`. It feels kinda rare to use this and I don't see it [publicly used anywhere](https://github.com/search?q=import+AwaitWriteFinishOptions+vite+NOT+is%3Afork&type=code) on github. So I figured to remove this.